packages/timeline/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@twreporter/timeline",
+  "version": "1.0.0",
+  "description": "The Reporter timeline component",
+  "main": "lib/index.js",
+  "repository": "https://github.com/twreporter/orangutan.git",
+  "author": "twreporter <developer@twreporter.org>",
+  "license": "MIT",
+  "scripts": {
+    "clean": "make clean",
+    "build": "make build",
+    "dev": "make dev",
+    "prepublishOnly": "make build",
+    "dev-server": "webpack-dev-server"
+  },
+  "devDependencies": {
+    "webpack-dev-server": "^3.7.2",
```

Comment:
`webpack-dev-server` is added in the root package.json

Reply:
I need to add it for telling yarn to create `.bin` in the `packages/timeline/node_modules`  to run `webpack-dev-server` at `packages/timeline`. Or should I use absolute path in npm scripts instead?

Reply:
How about follow this template?
https://github.com/twreporter/orangutan-monorepo/blob/d003d5b35e0657a2301ed5a6b802d8cdb49dfb2b/packages/dual-channel/Makefile#L11-L15

in package.json, 
```
scripts: {
  "dev-server": "make dev-server",
}
```